src/odemis/gui/util/widgets.py

```diff
@@ -126,8 +126,9 @@ def _connect(self, init):
 
     def disconnect(self):
         logging.debug("Disconnecting VigilantAttributeConnector")
-        for event in self.change_events:
-            self.value_ctrl.Unbind(event, source=self.value_ctrl, handler=self._on_ctrl_value_change)
+        if self.value_ctrl:
```

Comment:
Indeed, it is, as disconnect() is called only from destroy(). However, I wanted to make sure it's possible to call disconnect() from other places, for the future.
Alternatively, we could just merge disconnect() into destroy(), and get rid of the extra code. But let's keep it complicated for now ;-)